include/godot_cpp/variant/variant.hpp

```diff
@@ -211,6 +212,7 @@ class Variant {
 	Variant(const PackedStringArray &v);
 	Variant(const PackedVector2Array &v);
 	Variant(const PackedVector3Array &v);
+	Variant(const PackedVector4Array &v);
```

Comment:
This should be after PackedColorArray, to stay consistent with the inconsistent order of `Variant::Type`. Same below.